skupper/network/roles/skupper_init/tasks/kube-tasks.yml

```diff
@@ -4,6 +4,10 @@
 - name: Prepare init command
   ansible.builtin.include_tasks: kube-command.yml
 
+- name: Log init command
+  ansible.builtin.debug:
+    msg: running {{ skupper_init_command }}  
```

Comment:
I would just recommend doing this at: `skupper_init/tasks/main.yml`, before the actual execution,
so that it is done for podman sites as well.